pkg/controller/jobs/jobset/jobset_webhook.go

```diff
@@ -75,12 +75,13 @@ func (w *JobSetWebhook) Default(ctx context.Context, obj runtime.Object) error {
 		}
 		clusterQueueName, err := w.queues.ClusterQueueFromLocalQueue(queue.QueueKey(jobSet.ObjectMeta.Namespace, localQueueName))
 		if err != nil {
-			return err
+			log.V(5).Info("Failed to get cluster queue name", "jobset", klog.KObj(jobSet), "error", err)
+			return nil
 		}
 		for _, admissionCheck := range w.cache.AdmissionChecksForClusterQueue(clusterQueueName) {
-			if admissionCheck.Controller == multikueue.ControllerName {
-				log.V(5).Info("Defaulting ManagedBy", "jobset", klog.KObj(jobSet), "oldManagedBy", jobSet.Spec.ManagedBy, "managedBy", multikueue.ControllerName)
-				jobSet.Spec.ManagedBy = ptr.To(multikueue.ControllerName)
+			if admissionCheck.Controller == v1alpha1.MultiKueueControllerName {
```

Comment:
Please de-couple the JobSet fix to a separate PR. We may want to cherry-pick it onto 0.7 (not confirmed yet, but let's keep the decisions about cherry-picking independent).

Reply:
+1